Creational/README.rst

```diff
@@ -6,7 +7,7 @@ suitable to the situation. The basic form of object creation could result in
 design problems or added complexity to the design. Creational design patterns
 solve this problem by somehow controlling this object creation.
 
-* [AbstractFactory](AbstractFactory) [:notebook:](http://en.wikipedia.org/wiki/Abstract_factory_pattern)
+* :doc:`AbstractFactory <AbstractFactory/README>` `Wikipedia <http://en.wikipedia.org/wiki/Abstract_factory_pattern>`_
```

Comment:
@eddiejaoude did you investigate and found the reason? I would really like to merge this in :+1: